### ISSUE TITLE
Example of handleTriggerClick please

### ISSUE THREAD

**Issue by DavidSabine** [glennflanagan/react-collapsible#150]
**Use Case**

I'm using `<Collapsible/>` within a form.

I like that it can be triggered by `Enter` or `Space` (for accessibility reasons) but I don't like that it can be triggered without having focus. (i.e. Pressing `Enter` while in any of the form elements (which are children of `<Collapsible/>` will trigger the accordion.)

In my case, I actually want the accordion to never close after its been opened. So, I'm trying to override handleTriggerClick and prevent the closing of the accordion.

**What I've Tried**

```
        <Collapsible trigger={'Hello'}
        handleTriggerClick={(event) => { alert(typeof(event)) }}>
          <p>World</p>
        </Collapsible>
```

Steps to reproduce my experiment:

1. Go go [React Collapsible JSFiddle Template](https://jsfiddle.net/982f3LL7)
2. Replace the <Collapsible/> component in the Fiddle with my code above.
3. Click `Run`
4. Note that `event` is undefined.

5. Try `handleTriggerClick={() => { alert(typeof(this)) }}`
6. Try `props`?

**Expected behavior**
Frankly, I'm not sure what to expect. I think I should be able to provide a new handleTriggerClick function and have access to `this` and `this.props` or the `event` object.

Basically, I want to access the `open` state of the component and, if already open, I want to prevent the closing of the accordion. (With `triggerOpenedClassName`, I can 'hide' the trigger element but the `Enter` and `Space` keys still trigger the element.)

I suspect it's easy to do. None of the examples in the Readme or in the Issues queue has led me to the answer. Any help to get me going in the right direction would be appreciated.

**Comment by glennflanagan** [comment]
If you want to be able to access the open state from outside of the collapsible component then I recommend managing the `open` prop in the surrounding component which will allow you to determine if it's open or not.

**Comment by wsaun-edh** [comment]
I agree with @DavidSabine
